src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloud.java

```diff
@@ -223,9 +223,9 @@ private synchronized Integer availableNodeCapacity() throws IOException {
         }
     }
 
-    private synchronized ComputeEngineInstance getAnAgent(InstanceConfiguration config, Label requiredLabel) {
+    private synchronized ComputeEngineInstance getAnAgent(InstanceConfiguration config, String labels) {
```

Comment:
Is this method dead code? It doesn't seem to be called from anywhere.

Reply:
Seems that way..

But it was there previously so it should not be removed in this PR but rather extra PR for cleaning..

Reply:
In that case, an issue for the re-factor should be created and linked from here in a comment.